public/theme/boltcms-2021/partials/_cta.twig

```diff
@@ -0,0 +1,15 @@
+{# Start of "CTA" section #}
```

Comment:
Is this used now that we have `_bottom-block.twig`?

Reply:
No

Reply:
Let's get rid of it then 😉

Reply:
Or, what Bob would say: Kill it with fire 🔥

Reply:
Done!